---
#layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

## Peer-Reviewed Publication

\* indicates equal contribution

**Evaluation of Data-Dependent MS/MS Acquisition Parameters for Non-Targeted Metabolomics and Molecular Networking of Environmental Samples: Focus on the Q Exactive Platform**
Stincone P, Pakkir Shah AK, Schmid R, Graves LG, Lambidis SP, Torres RR, Xia S-N, Minda V, Aron AT, Wang M, Hughes CC, Petras D    
Analytical chemistry. 2023;95: 12673–12682. doi:10.1021/acs.analchem.3c01202

**Integrative analysis of multimodal mass spectrometry data in MZmine 3**
Schmid R*, Heuckeroth S*, Korf A*, Smirnov A, Myers O, Dyrlund TS, Bushuiev R, Murray KJ, Hoffmann N, Lu M, Sarvepalli A, Zhang Z, Fleischauer M, Dührkop K, Wesner M, Hoogstra SJ, Rudt E, Mokshyna O, Brungs C, Ponomarov K, Mutabdžija L, Damiani T, Pudney CJ, Earll M, Helmer PO, Fallon TR, Schulze T, Rivas-Ubach A, Bilbao A, Richter H, Nothias L-F, Wang M, Orešič M, Weng J-K, Böcker S, Jeibmann A, Hayen H, Karst U, Dorrestein PC, Petras D, Du X, Pluskal T    
Nature biotechnology. 2023;41: 447–449. doi:10.1038/s41587-023-01690-2

**Comparison of Cosine, Modified Cosine, and Neutral Loss Based Spectrum Alignment For Discovery of Structurally Related Molecules**
Bittremieux W*, Schmid R*, Huber F, van der Hooft JJJ, Wang M, Dorrestein PC    
Journal of the American Society for Mass Spectrometry. 2022;33: 1733–1744. doi:10.1021/jasms.2c00153

**Tattoo Pigment Identification in Inks and Skin Biopsies of Adverse Reactions by Complementary Elemental and Molecular Bioimaging with Mass Spectral Library Matching**
Brungs C*, Schmid R*, Wolf C, Berg T, Korf A, Heuckeroth S, Hayen H, van der Bent S, Maijer K, Rustemeyer T, Karst U    
Analytical chemistry. 2022;94: 3581–3589. doi:10.1021/acs.analchem.1c04922

**foodMASST a mass spectrometry search tool for foods and beverages**
West KA, Schmid R, Gauglitz JM, Wang M, Dorrestein PC 
NPJ science of food. 2022;6: 22. doi:10.1038/s41538-022-00137-3
GNPS Dashboard: collaborative exploration of mass spectrometry data in the web browser 
Petras D, Phelan VV, Acharya D, Allen AE, Aron AT, Bandeira N, Bowen BP, Belle-Oudry D, Boecker S, Cummings DA Jr, Deutsch JM, Fahy E, Garg N, Gregor R, Handelsman J, Navarro-Hoyos M, Jarmusch AK, Jarmusch SA, Louie K, Maloney KN, Marty MT, Meijler MM, Mizrahi I, Neve RL, Northen TR, Molina-Santiago C, Panitchpakdi M, Pullman B, Puri AW, Schmid R, Subramaniam S, Thukral M, Vasquez-Castro F, Dorrestein PC, Wang M    
Nature methods. 2022;19: 134–136. doi:10.1038/s41592-021-01339-5

**Native mass spectrometry-based metabolomics identifies metal-binding compounds**
Aron AT, Petras D, Schmid R, Gauglitz JM, Büttel I, Antelo L, Zhi H, Nuccio S-P, Saak CC, Malarney KP, Thines E, Dutton RJ, Aluwihare LI, Raffatellu M, Dorrestein PC     
Nature chemistry. 2022;14: 100–109. doi:10.1038/s41557-021-00803-1

**Ion identity molecular networking for mass spectrometry-based metabolomics in the GNPS environment**
Schmid R*, Petras D*, Nothias L-F*, Wang M, Aron AT, Jagels A, Tsugawa H, Rainer J, Garcia-Aloy M, Dührkop K, Korf A, Pluskal T, Kameník Z, Jarmusch AK, Caraballo-Rodríguez AM, Weldon KC, Nothias-Esposito M, Aksenov AA, Bauermeister A, Albarracin Orio A, Grundmann CO, Vargas F, Koester I, Gauglitz JM, Gentry EC, Hövelmann Y, Kalinina SA, Pendergraft MA, Panitchpakdi M, Tehan R, Le Gouellec A, Aleti G, Mannochio Russo H, Arndt B, Hübner F, Hayen H, Zhi H, Raffatellu M, Prather KA, Aluwihare LI, Böcker S, McPhail KL, Humpf H-U, Karst U, Dorrestein PC     
Nature communications. 2021;12: 3832. doi:10.1038/s41467-021-23953-9

**Feature-based molecular networking for identification of organic micropollutants including metabolites by non-target analysis applied to riverbank filtration**
Oberleitner D, Schmid R, Schulz W, Bergmann A, Achten C     
Analytical and bioanalytical chemistry. 2021;413: 5291–5300. doi:10.1007/s00216-021-03500-7

**Auto-deconvolution and molecular networking of gas chromatography-mass spectrometry data**
Aksenov AA, Laponogov I, Zhang Z, Doran SLF, Belluomo I, Veselkov D, Bittremieux W, Nothias LF, Nothias-Esposito M, Maloney KN, Misra BB, Melnik AV, Smirnov A, Du X, Jones KL 2nd, Dorrestein K, Panitchpakdi M, Ernst M, van der Hooft JJJ, Gonzalez M, Carazzone C, Amézquita A, Callewaert C, Morton JT, Quinn RA, Bouslimani A, Orio AA, Petras D, Smania AM, Couvillion SP, Burnet MC, Nicora CD, Zink E, Metz TO, Artaev V, Humston-Fulmer E, Gregor R, Meijler MM, Mizrahi I, Eyal S, Anderson B, Dutton R, Lugan R, Boulch PL, Guitton Y, Prevost S, Poirier A, Dervilly G, Le Bizec B, Fait A, Persi NS, Song C, Gashu K, Coras R, Guma M, Manasson J, Scher JU, Barupal DK, Alseekh S, Fernie AR, Mirnezami R, Vasiliou V, Schmid R, Borisov RS, Kulikova LN, Knight R, Wang M, Hanna GB, Dorrestein PC, Veselkov K      
Nature biotechnology. 2021;39: 169–173. doi:10.1038/s41587-020-0700-3

**Spatially and size-resolved analysis of gold nanoparticles in rat spleen after intratracheal instillation by laser ablation-inductively coupled plasma-mass spectrometry**
Nordhorn ID, Dietrich D, Verlemann C, Vennemann A, Schmid R, Elinkmann M, Fuchs J, Sperling M, Wiemann M, Karst U     
Metallomics. 2021;13. doi:10.1093/mtomcs/mfab028

**Feature-based molecular networking in the GNPS analysis environment**
Nothias L-F*, Petras D*, Schmid R*, Dührkop K, Rainer J, Sarvepalli A, Protsyuk I, Ernst M, Tsugawa H, Fleischauer M, Aicheler F, Aksenov AA, Alka O, Allard P-M, Barsch A, Cachet X, Caraballo-Rodriguez AM, Da Silva RR, Dang T, Garg N, Gauglitz JM, Gurevich A, Isaac G, Jarmusch AK, Kameník Z, Kang KB, Kessler N, Koester I, Korf A, Le Gouellec A, Ludwig M, Martin H C, McCall L-I, McSayles J, Meyer SW, Mohimani H, Morsy M, Moyne O, Neumann S, Neuweger H, Nguyen NH, Nothias-Esposito M, Paolini J, Phelan VV, Pluskal T, Quinn RA, Rogers S, Shrestha B, Tripathi A, van der Hooft JJJ, Vargas F, Weldon KC, Witting M, Yang H, Zhang Z, Zubeil F, Kohlbacher O, Böcker S, Alexandrov T, Bandeira N, Wang M, Dorrestein PC     
Nature methods. 2020;17: 905–908. doi:10.1038/s41592-020-0933-6

**Reproducible molecular networking of untargeted mass spectrometry data using GNPS**
Aron AT, Gentry EC, McPhail KL, Nothias L-F, Nothias-Esposito M, Bouslimani A, Petras D, Gauglitz JM, Sikora N, Vargas F, van der Hooft JJJ, Ernst M, Kang KB, Aceves CM, Caraballo-Rodríguez AM, Koester I, Weldon KC, Bertrand S, Roullier C, Sun K, Tehan RM, Boya P CA, Christian MH, Gutiérrez M, Ulloa AM, Tejeda Mora JA, Mojica-Flores R, Lakey-Beitia J, Vásquez-Chaves V, Zhang Y, Calderón AI, Tayler N, Keyzers RA, Tugizimana F, Ndlovu N, Aksenov AA, Jarmusch AK, Schmid R, Truman AW, Bandeira N, Wang M, Dorrestein PC     
Nature protocols. 2020;15: 1954–1991. doi:10.1038/s41596-020-0317-5

**Identification of potential human urinary biomarkers for tomato juice intake by mass spectrometry-based metabolomics**
Hövelmann Y, Jagels A, Schmid R, Hübner F, Humpf H-U     
European journal of nutrition. 2020;59: 685–697. doi:10.1007/s00394-019-01935-4

**Digging deeper - A new data mining workflow for improved processing and interpretation of high resolution GC-Q-TOF MS data in archaeological research**
Korf A, Hammann S, Schmid R, Froning M, Hayen H, Cramp LJE    
Scientific reports. 2020;10: 767. doi:10.1038/s41598-019-57154-8

**Fast Online Separation and Identification of Electrochemically Generated Isomeric Oxidation Products by Trapped Ion Mobility–Mass Spectrometry**
Fangmeyer J, Scheeren SG, Schmid R, Karst U    
Analytical chemistry. 2020;92: 1205–1210. doi:10.1021/acs.analchem.9b04337

**Expanding the Kendrick Mass Plot Toolbox in MZmine 2 to Enable Rapid Polymer Characterization in Liquid Chromatography-Mass Spectrometry Data Sets**
Korf A, Fouquet T, Schmid R, Hayen H, Hagenhoff S    
Analytical chemistry. 2020;92: 628–633. doi:10.1021/acs.analchem.9b03863

**Multimodal imaging of hallucinogens 25C- and 25I-NBOMe on blotter papers**
Lützen E, Holtkamp M, Stamme I, Schmid R, Sperling M, Pütz M, Karst U     
Drug testing and analysis. 2020;12: 465–471. doi:10.1002/dta.2751

**Lipid Species Annotation at Double Bond Position Level with Custom Databases by Extension of the MZmine 2 Open-Source Software Package**
Korf A, Jeck V, Schmid R, Helmer PO, Hayen H     
Analytical chemistry. 2019;91: 5098–5105. doi:10.1021/acs.analchem.8b05493

**Three-dimensional Kendrick mass plots as a tool for graphical lipid identification**
Korf A, Vosse C, Schmid R, Helmer PO, Jeck V, Hayen H     
Rapid communications in mass spectrometry. 2018;32: 981–991. doi:10.1002/rcm.8117

**A Fungal N-Dimethylallyltryptophan Metabolite from Fusarium fujikuroi**
Arndt B, Janevska S, Schmid R, Hübner F, Tudzynski B, Humpf H-U     
Chembiochem. 2017;18: 899–904. doi:10.1002/cbic.201600691

**Microfluidic high performance liquid chromatography-chip hyphenation to inductively coupled plasma-mass spectrometry**
Bishop DP, Blanes L, Wilson AB, Wilbanks T, Killeen K, Grimm R, Wenzel R, Major D, Macka M, Clarke D, Schmid R, Cole N, Doble PA     
Journal of chromatography A. 2017;1497: 64–69. doi:10.1016/j.chroma.2017.03.025





## Preprints

# indicates corresponding author * indicates equal contribution
**On-tissue dataset-dependent MALDI-TIMS-MS2 bioimaging**
Heuckeroth S, Behrens A, Wolf C, Fütterer A, Nordhorn ID, Kronenberg K, Brungs C, Korf A, Richter H, Jeibmann A, Karst U, Schmid R#     
ChemRxiv. 2023. doi:10.26434/chemrxiv-2023-nw5p3-v2

**A Taxonomically-informed Mass Spectrometry Search Tool for Microbial Metabolomics Data**
Zuffa S*, Schmid R*, Bauermeister A, Gomes PWP, Caraballo-Rodriguez AM, El Abiead Y, Aron AT, Gentry EC, Zemlin J, Meehan MJ, Avalon NE, Cichewicz RH, Buzun E, Terrazas MC, Hsu C-Y, Oles R, Ayala AV, Zhao J, Chu H, Kuijpers MCM, Jackrel SL, Tugizimana F, Nephali LP, Dubery IA, Madala NE, Moreira EA, Costa-Lotufo LV, Lopes NP, Rezende-Teixeira P, Jimenez PC, Rimal B, Patterson AD, Traxler MF, de Cassia Pessotti R, Alvarado-Villalobos D, Tamayo-Castillo G, Chaverri P, Escudero-Leyva E, Quiros-Guerrero L-M, Bory AJ, Joubert J, Rutz A, Wolfender J-L, Allard P-M, Sichert A, Pontrelli S, Pullman BS, Bandeira N, Gerwick WH, Gindro K, Massana-Codina J, Wagner BC, Forchhammer K, Petras D, Aiosa N, Garg N, Liebeke M, Bourceau P, Kang KB, Gadhavi H, de Carvalho LPS, dos Santos MS, Pérez-Lorente AI, Molina-Santiago C, Romero D, Franke R, Brönstrup M, de León AVP, Pope PB, La Rosa SL, La Barbera G, Roager HM, Laursen MF, Hammerle F, Siewert B, Peintner U, Licona-Cassani C, Rodriguez-Orduña L, Rampler E, Hildebrand F, Koellensperger G, Schoeny H, Hohenwallner K, Panzenboeck L, Gregor R, O'Neill EC, Roxborough ET, Odoi J, Bale NJ, Ding S, Sinninghe Damsté JS, Guan XL, Cui JJ, Ju K-S, Silva DB, Silva FMR, da Silva GF, Koolen HHF, Grundmann C, Clement JA, Mohimani H, Broders K, McPhail KL, Ober-Singleton SE, Rath CM, McDonald D, Knight R, Wang M, Dorrestein PC     
bioRxiv. 2023. p. 2023.07.20.549584. doi:10.1101/2023.07.20.549584

**Open Access Repository-Scale Propagated Nearest Neighbor Suspect Spectral Library for Untargeted Metabolomics**
Bittremieux W, Avalon NE, Thomas SP, Kakhkhorov SA, Aksenov AA, Gomes PWP, Aceves CM, Caraballo-Rodríguez AM, Gauglitz JM, Gerwick WH, Huan T, Jarmusch AK, Kaddurah-Daouk RF, Kang KB, Kim HW, Kondić T, Mannochio-Russo H, Meehan MJ, Melnik AV, Nothias L-F, O'Donovan C, Panitchpakdi M, Petras D, Schmid R, Schymanski EL, van der Hooft JJJ, Weldon KC, Yang H, Xing S, Zemlin J, Wang M, Dorrestein PC     
bioRxiv. 2023. p. 2022.05.15.490691. doi:10.1101/2022.05.15.490691

**The Underappreciated Diversity of Bile Acid Modifications**
Mohanty I, Mannochio-Russo H, El Abiead Y, Schweer JV, Bittremieux W, Xing S, Schmid R, Zuffa S, Vasquez F, Muti VB, Zemlin J, Tovar-Herrera OE, Moraïs S, Desai D, Amin S, Koo I, Turck CW, Mizrahi I, Huan T, Patterson AD, Siegel D, Hagey LR, Wang M, Aron AT, Dorrestein P    
SSRN, 2023. doi:10.2139/ssrn.4436846

**A multi-omics strategy for the study of microbial metabolism: application to the human skin's microbiome**
Nothias L-F, Schmid R, Garlet A, Cameron H, Leoty-Okombi S, André-Frei V, Fuchs R, Dorrestein PC, Ternes P    
bioRxiv. 2023. doi:10.1101/2023.03.26.532286

**Two-Dimensional Liquid Chromatography Tandem-Mass Spectrometry Untangles the Deep Metabolome of Marine Dissolved Organic Matter**
Lambidis SP, Schramm T, Steuer-Lodd K, Farrell S, Stincone P, Schmid R, Koester I, Torres R, Aluwihare L, Simon C, Petras D     
ChemRxiv. 2023. doi:10.26434/chemrxiv-2023-j1bxh





## Book Chapters

**Processing Metabolomics and Proteomics Data with Open Software: A Practical Guide**
Chapter 7: Metabolomics Data Analysis Using MZmine
Pluskal T, Korf A, Smirnov A, Schmid R, Fallon TR, Du X, Weng J-K    
Royal Society of Chemistry, 2020, pp. 232–254. 
